slf4j/src/main/scala/io/odin/slf4j/Slf4jLogger.scala

```diff
@@ -23,16 +23,20 @@ import io.odin.{Level, Logger, LoggerMessage}
 import org.slf4j.{Logger => JLogger, LoggerFactory}
 import cats.implicits._
 
-final class Slf4jLogger[F[_]: Sync](logger: JLogger, level: Level, formatter: Formatter)
-    extends DefaultLogger[F](level) {
+final class Slf4jLogger[F[_]: Sync](
+    logger: JLogger,
+    level: Level,
+    formatter: Formatter,
+    syncType: Sync.Type = Sync.Type.Blocking
```

Comment:
Same as `ConsoleLogger`, I think it's better the constructor in the companion object.